css_stylesheet.css

```diff
@@ -56,7 +55,8 @@ header {
 	font-family: "Courier New", serif;
 	color: #303030;
 	float: right;
-	margin-top: 55px;
+	margin-top: 2em;
+	font-size: 1.9em;;
```

Comment:
You've got two semicolons on line 59. I don't think it has any functional implications, but the second one should still be removed

Reply:
Fixed, Thanks